_posts/2020-03-02-tomcat-thread-model.md

```diff
@@ -11,42 +11,42 @@ tags:
-* [Service](http://tomcat.apache.org/tomcat-7.0-doc/config/service.html) - Zawiera konfigurację poszczególnych connectorów łącząc je silnikiem przetwarzania żądań.
+* [Service](http://tomcat.apache.org/tomcat-7.0-doc/config/service.html) - zawiera konfigurację poszczególnych connectorów łącząc je silnikiem przetwarzania żądań.
 * [Connector](http://tomcat.apache.org/tomcat-7.0-doc/config/http.html) - nasłuchuje na wybranym porcie http i obsługuje połączenia przekazując żądania do silnika zdefiniowanego w elemencie Engine.
 * [Engine](http://tomcat.apache.org/tomcat-7.0-doc/config/engine.html) - przetwarza żądania pochodzące ze wszystkich connectorów zdefiniowanych w elemencie Service.
 
-Z punktu widzenia przetwarzania żądań kluczowe znaczenie ma konfiguracja connectora i parametry:
-* protocol - pozwala określić wybraną implementacje connectora (to tutaj możemy zdecydować czy chcemy użyć connectora blokującego czy nieblokującego).
-* acceptCount - długość kolejki oczekujących połączeń. Kolejka napełnia się jeżeli wszystkie wątki w puli wątków connectora są zajęte. W przypadku osiągnięcia limitu tej kolejki (domyślnie 100) połączenia klientów będą odrzucane.
-* maxConnections - maksymalna liczba połączeń, które mogą być przetwarzane. Po osiągnięciu maksymalnej liczby połączeń, połączenia nadal są przyjmowane i kolejkowane do czasu osiągnięcia limitu wynikającego z parametru acceptCount. Domyślnie maxConnections jest równe maxThreads dla connectora blokującego i 10000 dla nieblokującego.
-* maxThreads - maksymalna liczba wątków obsługujących żądania. Parametr ten oznacza ile żądań może być symultanicznie przetwarzanych przez serwer. Domyślna wartość tego parametru to 200. Biorąc pod uwagę problem, którym zajmujemy się w tym artykule jest to najważniejszy parametr.
+Z punktu widzenia przetwarzania żądań, kluczowe znaczenie ma konfiguracja connectora i parametry:
+* protocol - Pozwala określić wybraną implementacje connectora (to tutaj możemy zdecydować czy chcemy użyć connectora blokującego czy nieblokującego).
+* acceptCount - Długość kolejki oczekujących połączeń. Kolejka napełnia się jeżeli wszystkie wątki w puli wątków connectora są zajęte. W przypadku osiągnięcia limitu tej kolejki (domyślnie 100) połączenia klientów będą odrzucane.
+* maxConnections - Maksymalna liczba połączeń, które mogą być przetwarzane. Po osiągnięciu maksymalnej liczby połączeń, połączenia nadal są przyjmowane i kolejkowane do czasu osiągnięcia limitu wynikającego z parametru acceptCount. Domyślnie maxConnections jest równe maxThreads dla connectora blokującego i 10000 dla nieblokującego.
+* maxThreads - Maksymalna liczba wątków obsługujących żądania. Parametr ten oznacza, ile żądań może być symultanicznie przetwarzanych przez serwer. Domyślna wartość tego parametru to 200. Jest to najważniejszy parametr, zważywszy na problem, który rozwiązujemy w tym artykule.
 
```

Comment:
z czego wynika tutejsza zmiana na dużą literę? Moim zdaniem powinno być tak samo jak powyżej, czyli po myślniku mamy małą literę. Jednocześnie w takich przpadkach, wyliczenia kończyłbym średnikiem, a dopiero po ostatnim punkcie, wstawiłbym kropkę.
https://popolskiemu.pl/jak-sporzadzac-wyliczenia/
Pokusa stawiania kropki jest spora, w końcu to długie zdania, ale jednak wszystkie są poprzedzone konkretnym wstępem (anonsujesz, że będą przedstawiane parametry).

Reply:
W jednym i drugim linku, który podałeś nie ma przypadku kiedy konkretna nazwa przypadku plus długie zdania. Nie mam moim zdaniem ściśle zdefiniowanych reguł, a to co jest teraz wygląda moim zdaniem lepiej niż, np:
* maxConections - Maksymalna liczba połączeń, które mogą być przetwarzane. Po osiągnięciu maksymalnej liczby połączeń, połączenia nadal są przyjmowane i kolejkowane do czasu osiągnięcia limitu wynikającego z parametru acceptCount. Domyślnie maxConnections jest równe maxThreads dla connectora blokującego i 10000 dla nieblokującego**;**

Nie może być tak, że jest 5 zdań, wszystkie zakończone kropką i nagle na końcu ostatniego jest średnik.

Reply:
Masz rację, to mnie wewnętrznie przekonuje.